### DISCUSSION TITLE
Podman support

### DISCUSSION THREAD

**Discussion by elf55** [IceWhaleTech/CasaOS#676] [Ideas]
I've just started using CasaOS and it punches a lot of buttons that I need.  Excellent work and thank you.  
I see that this has been suggested before, but please consider support for Podman in lieu of Docker.  Aside from the security advantages of Podman, the command structure of Podman is the same as Docker and there is a compatible if not identical API.  I've been switching all my work to Podman from Docker and so far haven't had any significant issues, including with docker compose.  

I think CasaOS would be a stronger and more secure product if Podman was the container agent.

**Comment by yieme** [comment]
I agree with security needs noted in #47 and see that it was closed as "completed". If it is completed, then how might we reconfigure CasaOS to use Podman? If not, I would like to echo @elf55 and request that you reconsider for the sake of concern for the security of the Home user. Thanks :-)
